tests/integration/run_tests.rs

```diff
@@ -515,7 +515,7 @@ fn _090_run_permissions_request() {
         "┠─ Requested by `Deno.permissions.request()` API.\r\n",
         "┠─ Learn more at: https://docs.deno.com/go/--allow-run\r\n",
         "┠─ Run again with --allow-run to bypass this prompt.\r\n",
-        "┗ Allow? [y/n/A] (y = yes, allow; n = no, deny; A = allow all run permissions)",
+        "┗ Allow? [y/n/A] (y = yes, allow; n = no, deny; A = allow all run permissions; D = deny all run permissions)",
```

Comment:
This makes this line go beyond 80 characters, which is not great. I'm not sure it's worth adding this.

Reply:
@dsherret it's a prompt message, if all this change not worth I will close the pr, please confirm if we need to change like this ?